src/server/search/search_family.cc

```diff
@@ -30,7 +30,7 @@ using namespace facade;
 
 namespace {
 
-unordered_map<string_view, search::Schema::FieldType> kSchemaTypes = {
+const unordered_map<string_view, search::Schema::FieldType> kSchemaTypes = {
```

Comment:
Any reason not to use `absl::flat_hash_map`?
I realize it's a small map, still I think it's better